### ISSUE TITLE
5.0+ frameworks do not install

### ISSUE THREAD

**Issue by matejdro** [Furniel/Apk-Changer#4]
```
        at brut.androlib.res.AndrolibResources.installFramework(AndrolibResources
.java:611)
        at brut.androlib.Androlib.installFramework(Androlib.java:469)
        at brut.apktool.Main.cmdInstallFramework(Main.java:253)
        at brut.apktool.Main.main(Main.java:81)
Caused by: java.io.IOException: Int Expected: 0x001c0001, got: 0x00000000
        at brut.util.ExtDataInput.skipCheckInt(ExtDataInput.java:48)
        at brut.androlib.res.decoder.StringBlock.read(StringBlock.java:43)
        at brut.androlib.res.decoder.ARSCDecoder.readPackage(ARSCDecoder.java:100
)
        at brut.androlib.res.decoder.ARSCDecoder.readTable(ARSCDecoder.java:81)
        at brut.androlib.res.decoder.ARSCDecoder.decode(ARSCDecoder.java:49)
        ... 5 more
Exception in thread "main" brut.androlib.AndrolibException: Could not decode arsc
 file
        at brut.androlib.res.decoder.ARSCDecoder.decode(ARSCDecoder.java:54)
        at brut.androlib.res.AndrolibResources.getResPackagesFromApk(AndrolibReso
urces.java:540)
        at brut.androlib.res.AndrolibResources.loadMainPkg(AndrolibResources.java
:76)
        at brut.androlib.res.AndrolibResources.getResTable(AndrolibResources.java
:68)
        at brut.androlib.Androlib.getResTable(Androlib.java:51)
        at brut.androlib.ApkDecoder.getResTable(ApkDecoder.java:191)
        at brut.androlib.ApkDecoder.decode(ApkDecoder.java:116)
        at brut.apktool.Main.cmdDecode(Main.java:146)
        at brut.apktool.Main.main(Main.java:77)
Caused by: java.io.IOException: Int Expected: 0x001c0001, got: 0x00000000
        at brut.util.ExtDataInput.skipCheckInt(ExtDataInput.java:48)
        at brut.androlib.res.decoder.StringBlock.read(StringBlock.java:43)
        at brut.androlib.res.decoder.ARSCDecoder.readPackage(ARSCDecoder.java:100
)
        at brut.androlib.res.decoder.ARSCDecoder.readTable(ARSCDecoder.java:81)
        at brut.androlib.res.decoder.ARSCDecoder.decode(ARSCDecoder.java:49)
        ... 8 more
Installing ANDROID 4.4w...
Installing ANDROID 4.4...
Installing ANDROID 4.3...
Installing ANDROID 4.2...
```

Picked all recommended settings. What am I doing wrong?

**Comment by matejdro** [comment]
I fixed issue by selecting latest apktool and smali rather than recommended one. Why is older one recommended then?

**Comment by Furniel** [comment]
New apktool is in active development now, recently author released 2.0 version but it have few serious bugs. In apkchanger i use latest builds of apktool, but i can`t recommend it now, we should wait for fixing of few bugs and than it will be set as recommended. Today i`ll make few changes to apkchanger - this bug shouldn`t occur even if apktool 1.5.3 selected.
